examples/angular2/package.json

```diff
@@ -1,16 +1,32 @@
 {
   "private": true,
   "scripts": {
-    "dev": "tsc --watch"
-  },
-  "devDependencies": {
-    "typescript": "1.7.5"
+    "start": "tsc && concurrently \"tsc -w\" \"lite-server\" ",
+    "typings": "typings"
   },
   "dependencies": {
-    "angular2": "2.0.0-beta.0",
-    "systemjs": "0.19.6",
-    "rxjs": "5.0.0-beta.0",
+    "@angular/common": "~2.0.2",
+    "@angular/compiler": "~2.0.2",
+    "@angular/core": "~2.0.2",
+    "@angular/forms": "~2.0.2",
+    "@angular/http": "~2.0.2",
+    "@angular/platform-browser": "~2.0.2",
```

Comment:
Fyi, a lot of these have been bumped to 2.2.3 of late. Could we test this PR is definitely working with the latest?